### ISSUE TITLE
Is there an easy way to wrap around left and right?

### ISSUE THREAD

**Issue by grimace** [zyra/ionic2-super-tabs#38]
I couldn't find it in the documentation, but I'm sure I might have missed something.

**Comment by ihadeed** [comment]
Not sure what you mean by wrap around left or right

**Comment by grimace** [comment]
If I have 5 tabs and I get to the far left and swipe left, go to the one on the far right.  i.e.    tab5<-tab1   or tab5->tab1.     or like    5,0,1,2,3,4,5,0.

**Comment by ihadeed** [comment]
Sorry @grimace, I'm still not able to understand your question.

Could you please demonstrate it with a video? or provide an example?

**Comment by grimace** [comment]
Sorry for the confusion.  How about this?
Let's say you have 3 tabs;  Tab1, Tab2, Tab3.
1. Starting from the left ( Tab1 )
2. Swipe right to Tab2
3. Swipe right to Tab3
4. Continue to swipe right from Tab3 - currently, nothing happens  ( of course you could go back the other way, but not what I was looking for )
5. What I'm interested in is the ability to swipe right from Tab3 ( wrapping back around ) to Tab1.  So that you could actually continuously swipe in one direction forever ( or until you get tired of the exercise).

**Comment by ihadeed** [comment]
@grimace 

oh, so like  the `loop` option on `ion-slides` ?

There isn't an option for that right now. I should be able to implement it in the near future.

**Comment by darron1217** [comment]
@ihadeed I'm expecting for `loop` option too!
Anyway, thank you for this beautiful plugin :)

**Comment by grimace** [comment]
Yes, exactly what I was looking for. Also like to express my gratitude. Great project. Thanks!

**Comment by NgYueHong** [comment]
Really hope to see this feature implemented soon.

**Comment by ihadeed** [comment]
@NgYueHong I'll try to implement this if it performs well. The new super tabs is based on a flex box .. so it should be easier to move the tabs around and give the user the feeling of an infinite loop.

**Comment by RajabMohammadi** [comment]
@ihadeed 
I work on ionic 4 application 
but when I loop the super-tab-button and super-tab 
I have this error : 
`core.js:15723 ERROR Error: Uncaught (in promise): TypeError: Cannot set property 'active' of undefined
TypeError: Cannot set property 'active' of undefined
    at SuperTabsToolbarComponent.push../node_modules/@ionic-super-tabs/core/dist/esm/legacy/super-tab-button_2.entry.js.SuperTabsToolbarComponent.markButtonActive (super-tab-button_2.entry.js:249)`

**Comment by NgYueHong** [comment]
@ihadeed It's great to see that Ionic 4 is now officially supported and I am really looking forward to see this feature implemented soon. With this feature, it will be a HUGE UPGRADE to this plugin that the plugin will be able to be used for infinite page swiping. Just like Gmail app, where user can just continuously swipe to view the next email.  Just imagine how much more useful this plugin will be with the implementation of infinite loop feature.

**Comment by Foxion7** [comment]
@ihadeed Is there any news on this?
